tests/mocks/world_mock.py

```diff
@@ -0,0 +1,7 @@
+import sys
+from tests.mocks import ui_mock
+
+def mock_methods(methods):
```

Comment:
This function is for mocking methods of the world module. This is useful because when we run the test we don't want to load the entire game from the ground up. Because of this some functions won't work correctly (e.g. graphic assets arent loaded) so we mock this functions in order to avoid errors and simplify what we want.